sdk/communication/communication-sms/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 1.1.0-beta.1 (Unreleased)
```

Comment:
Where do these SMS changes come from?

Reply:
The build is failing since the last release date is not within a month. So I'm just adding an entry for the next beta with unreleased to fix the build failures.

Reply:
Sounds good, thanks!